modules/dataquery/jsx/hooks/usebreadcrumbs.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useEffect} from 'react';
+import Breadcrumbs from 'jsx/Breadcrumbs';
+
+// Declared in smarty main.tpl
+declare const breadcrumbsRoot: any;
+declare const loris: any;
+
+/**
+ * Update the DQT breadcrumbs based on the active tab
+ *
+ * @param {string} activeTab - The active tab
+ * @param {function} setActiveTab - set the state on click
+ */
+function useBreadcrumbs(
+    activeTab: string,
+    setActiveTab: (newtab: string) => void
+) {
+    // update breadcrumbs breadcrumbs
+    useEffect(() => {
+        const breadcrumbs = [
+            {
+                text: 'Data Query Tool (Alpha)',
```

Comment:
I don't think that's easy to do since this is in TS and the Module class is in PHP. The work in doing it is probably much more significant than the gain. I think we should just keep them in sync when the "Alpha" gets removed from the description.